gnocchi/tests/functional/gabbits/resource.yaml

```diff
@@ -216,6 +216,15 @@ tests:
         $[1].revision_end: null
         $[1].metrics.'disk.iops': $RESPONSE["metrics.'disk.iops'"]
 
+    - name: get generic history with links
+      desc: Ensure we can get the history
+      xfail: true
```

Comment:
Why?

Reply:
I have added the oslo.db bug ref for each xfail